src/model_theory/definability.lean

```diff
@@ -45,6 +45,47 @@ structure definable (s : set (α → M)) : Prop :=
 
 variables {L} {A} {B : set M} {s : set (α → M)}
 
+lemma definable.map_expansion {L' : language} [L'.Structure M] (h : L.definable A s)
+  (φ : L →ᴸ L') [φ.is_expansion_on M] :
+  L'.definable A s :=
+begin
+  obtain ⟨ψ, rfl⟩ := h,
+  refine ⟨⟨(φ.add_constants A).on_formula ψ, _⟩⟩,
+  ext x,
+  simp only [mem_set_of_eq, Lhom.realize_on_formula],
+end
+
+lemma empty_definable_iff :
```

Comment:
```suggestion
lemma definable_empty_iff :
```
?

Reply:
The unfortunate thing is that `definable_empty` could mean two things, as there are two set arguments. Colloquially, I'm used to referring to a set definable over `A` as `A`-definable. Do you think it's worth putting this in the `set` namespace so I can type `A.definable L s` and thus clarify the word orders?

Reply:
If that matches your intended naming scheme better, that seems good to me.